ideas/README.md

```diff
@@ -0,0 +1,79 @@
+# Learning Ideas
+
+If you have some free time to learn by yourself, or in a pair, here are some ideas to help you use your time fruitfully.
+
+## Code Katas
+
+Go and do some Katas, either the ones [in this repository](../katas).
+
+Or from one of the following sources:
+
+* [Coding Dojo Kata](https://codingdojo.org/kata/)
+* [sleepyfox London Code Dojo](https://github.com/sleepyfox?utf8=%E2%9C%93&tab=repositories&q=dojo)
+* [Code Kata](http://codekata.com/)
+* [Code Katas](http://www.codekatas.org/)
+
+## Repeat Code Katas
+
+It goes without saying, but code katas are meant to be repeated.
+
+Each time, you should focus on honing a particular part of your skill set.
+
+You will pick up new knowledge as you finish workshops and seminars. 
+
+Focus on applying that new knowledge in practice.
+
+### Examples
+
+* Focus on getting names absolutely perfect, break out a thesaurus and choose the absolute most precise words for the problem at hand.
+* Is your code open for extension and closed for modification? Think of ways that it could be extended, and create plugin points
+* Do your classes and methods have one reason to change? Can you increase cohesion by refactoring?
+
+## Read
+
+Pick up a book and read, here are a few that are invaluable
+
+* Test Driven Development: By Example (Kent Beck)
+* Growing Object-Oriented Software Guided By Tests (Steve Freeman, Nat Pryce)
+* Domain Driven Design (Eric Evans)
+* The Pragmatic Programmer (Andy Hunt)
+* Clean Code (Robert C. Martin)
+* Refactoring (Martin Fowler)
+* The Clean Coder (Robert C. Martin)
+* Clean Architecture (Robert C. Martin)
+* Working Effectively with Legacy Code (Michael C. Feathers)
+* Practical Object Oriented Design in Ruby (Sandi Metz)
+* The Software Craftsman (Sandro Mancuso)
+
+## Watch
+
+Watch screencasts and other programming related videos
+
+* [Destroy All Software](https://www.destroyallsoftware.com/screencasts)
+* [Clean Coders](https://cleancoders.com/videos)
+* [Rails Casts](http://railscasts.com/)
+
+## Go back over workshops
+
+Can you apply the knowledge you picked up in a previous workshop or seminar in a new project.
+
+Do you need to follow the guide, or can you remember what to do?
+
+## Shadow others
+
+If possible, arrange to shadow a team actively developing software.
```

Comment:
This is a cool idea, is it worth letting them know which teams are open to this ahead of time? Be easier for them if let them know they're not gonna be intruding

Reply:
I've asked leads to invite the academy out for lunch, so a good time to ask I guess